extensions/blocks/map/component.js

```diff
@@ -115,8 +115,17 @@ export class Map extends Component {
 		this.debouncedSizeMap.cancel();
 	}
 	componentDidUpdate( prevProps ) {
-		const { apiKey, children, points, mapStyle, mapDetails, scrollToZoom } = this.props;
-		const { map } = this.state;
+		const {
+			apiKey,
+			children,
+			points,
+			mapStyle,
+			mapDetails,
+			scrollToZoom,
+			showFullscreenButton,
+			admin,
```

Comment:
Since we're making changes to this, do you think you could order those alphabetically? It's a nit-pick I know, but the `admin` in there confused me at first, and I think ordering those may help maybe?

Reply:
It's totally not a nitpick, and it's something I always do myself! 😄 
This time I think I've missed it because Prettier automatically expanded the object and I didn't notice.